src/lib/definitions.js

```diff
@@ -31,6 +22,10 @@ export function validateDefinition (definition, parserType) {
 }
 
 export async function parseDefinition ({ definition, parserType, navSort }) {
+  if (typeof definition === 'string') {
+    definition = YAML.safeLoad(definition)
```

Comment:
Question for another day, reckon the parser should be responsible for this? Or just be dumb and always expect a string?